### DISCUSSION TITLE
No /usr/share/taskwarrior directory was created during the install

### DISCUSSION THREAD

**Discussion by horror-vacui** [GothenburgBitFactory/taskwarrior#2436] [Community support] [answered]
Taskwarrior 2.5.3 has just been installed from the tarball downloaded from the taskwarrior website and it does not start by default:
```
zoltan@adrian ~ $ task
Could not read include file '/usr/share/taskwarrior/dark-256.theme'.
```

The command `zoltan@adrian /usr $ find ./ -iname "taskwarrior*"` does not return anything. 
I believe that the installation path might be different in ubuntu than on the system where taskwarrior has been tested. Unfortunately I am not savvy enough to find the installation location and point my .taskrc there.

I've uncommented the theme, and now task starts.

First of all I wanted to let you know, that something might be a bit off on Ubuntu 20.04.2 LTS, and I also hope that you could point me to the location where the themes (and probably other related stuff) could be.

**Comment by tbabej** [comment, marked as answer]
Hi @horror-vacui,

running the installation from the tarball in an Ubuntu container, I get the following output from the `make install`:

```
# make install | grep theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-yellow-green.theme
-- Up-to-date: /usr/local/share/doc/task/rc/no-color.theme
-- Up-to-date: /usr/local/share/doc/task/rc/solarized-light-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-blue-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-gray-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/light-16.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-16.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-violets-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-gray-blue-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/light-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/solarized-dark-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-green-256.theme
-- Up-to-date: /usr/local/share/doc/task/rc/dark-red-256.theme
```

I suspect you'll find the theme files there :)

**Comment by horror-vacui** [comment]
They were there. In hindsight I could have searched for the filename as well...
Thanks! :-)

**Comment by tbabej** [comment]
Glad to help. By the way, check out the discussions tab for future help requests!
